src/platform/intel/cavs/include/cavs/lib/asm_memory_management.h

```diff
@@ -25,7 +25,7 @@
 #include <sof/lib/memory.h>
 #include <sof/lib/shim.h>
 
-#if CAVS_VERSION >= CAVS_VERSION_1_8
+#if CAVS_VERSION >= CAVS_VERSION_1_8 || CONFIG_SOF_ZEPHYR
```

Comment:
The intention is to not use any platform, drivers or arch directories in the Zephyr build (as these will belong ion Zephyr). 

This Zephyr platform is temporary bridge to allow the incremental transition of code into Zephyr and to also allow development against platform code that is not yet public. This directly should shrink as code is moved to Zephyr and eventually disappear.

Reply:
I still see this naming as quite confusing, but since we are going to move this part to Zephyr and have plan to get rid of it then it probably can stay as is for now.

Reply:
@mwasko `CONFIG_SOF_ZEPHYR` is a new configuration option, that should be used for and platforms - cAVS or not - when building SOF with Zephyr. The goal is to move all architecture- and platform-specific code out of SOF to Zephyr for this configuration option. It will mean, that SOF is built as a pure Zephyr application with no platform / architecture code.
However, at the moment when building with Zephyr, SOF still uses architecture-specific code, including this header. So it has to be available both when building for cAVS, either with XTOS or with Zephyr, and when building in the generic Zephyr application mode, also for cAVS-compatible but strictly speaking non-cAVS platforms.